README.md

```diff
@@ -1,22 +1,32 @@
 # @hmcts/draft-store-client
 
+[![Travis badge](https://api.travis-ci.org/hmcts/draft-store-client.svg?branch=master)](https://travis-ci.org/hmcts/draft-store-client)
+
+[![Codecov badge](https://codecov.io/gh/hmcts/draft-store-client/graphs/badge.svg)](https://codecov.io/github/hmcts/draft-store-client)
+
+[![NPM version badge](https://img.shields.io/npm/v/@hmcts/draft-store-client.svg)](https://www.npmjs.com/@hmcts/draft-store-client)
+
+[![Node version badge](https://img.shields.io/node/v/@hmcts/draft-store-client.svg)](https://www.npmjs.com/@hmcts/draft-store-client)
+
```

Comment:
that's a lot of badges (6 in total), perhaps we could have them all in one line instead ? (like here for example: https://github.com/hmcts/draft-store)

Reply:
Great spot. Will do 👍